<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find loader_cont)/config/loader_control.yaml" command="load"/>

  <!-- load the controllers -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen" ns="/loader" args="joint5_velocity_controller joint1_velocity_controller joint2_velocity_controller joint3_velocity_controller joint4_velocity_controller joint_state_controller"/>

  <!-- convert joint states to TF transforms for rviz, etc -->
  <node name="robot_state_publisher" pkg="robot_state_publisher"   type="robot_state_publisher"
    respawn="false" output="screen">
    
  </node>

<node name="pub_control" pkg="loader_cont" type="control.py">
</node>


</launch>
